_includes/docker_platform_matrix.md

```diff
@@ -1,17 +1,17 @@
-| [Fedora](linux/fedora.md)                                                            |                   | {{ green-check }} |
-| [Oracle Linux](linux/oracle.md)                                                      | {{ green-check }} |                   |
-| [SUSE Linux Enterprise Server](linux/suse.md)                                        | {{ green-check }} |                   |
-| [Microsoft Windows Server 2016](https://docs.microsoft.com/en-us/virtualization/windowscontainers/quick-start/quick-start-windows-server){: target="_blank" class="_" } | {{ green-check }} |                   |
-| [Microsoft Windows 10](/docker-for-windows/))                                        |                   | {{ green-check }} |
-| [macOS](/docker-for-mac/)                                                            |                   | {{ green-check }} |
-| [Microsoft Azure](/docker-cloud/infrastructure/link-aws.md)                          | {{ green-check }} | {{ green-check }} |
-| [Amazon Web Services](/docker-cloud/infrastructure/link-aws.md)                      | {{ green-check }} | {{ green-check }} |
+| Platform                                                                                             | Docker EE         | Docker CE         |
+| ---------------------------------------------------------------------------------------------------- | ----------------- | ----------------- |
+| [Ubuntu](/engine/installation/linux/ubuntu.md)                                                       | {{ green-check }} | {{ green-check }} |
+| [Debian](/engine/installation/linux/debian.md)                                                       |                   | {{ green-check }} |
+| [Red Hat Enterprise Linux](/engine/installation/linux/rhel.md)                                       | {{ green-check }} |                   |
+| [CentOS](/engine/installation/linux/centos.md)                                                       | {{ green-check }} | {{ green-check }} |
+| [Fedora](/engine/installation/linux/fedora.md)                                                       |                   | {{ green-check }} |
+| [Oracle Linux](/engine/installation/linux/oracle.md)                                                 | {{ green-check }} |                   |
+| [SUSE Linux Enterprise Server](/engine/installation/linux/suse.md)                                   | {{ green-check }} |                   |
+| [Microsoft Windows Server 2016](https://docs.microsoft.com/en-us/virtualization/windowscontainers/quick-start/quick-start-windows-server){: target="_blank" class="_" } | {{ green-check }} |   |
+| [Microsoft Windows 10](/docker-for-windows/))                                                        |                   | {{ green-check }} |
+| [macOS](/docker-for-mac/)                                                                            |                   | {{ green-check }} |
```

Comment:
In linux we're linking directly to the install instructions. Maybe in `docker-for-mac` and `docker-for-windows` we should link to `.../install.md`, instead of just the index?